packages/botonic-react/tests/webchat/webchat.addmessages.test.jsx

```diff
@@ -24,7 +24,7 @@ describe('Adding webchat messageComponent', () => {
       )
     })
     await act(async () => {
-      render(<Webchat webchatHooks={result.current} />)
+      render(<Webchat host={document.body} webchatHooks={result.current} />)
```

Comment:
Why you have to add this `host` prop here?

Reply:
because now utils functions need the host as a parameter, otherwise tests would fail. Maybe I'll just add `document.body` as the default host inside Webchat component...